contrib/depends/hosts/freebsd.mk

```diff
@@ -0,0 +1,17 @@
+freebsd_CC=clang-8
```

Comment:
Does this only work for `clang-8`? Would be nice to get the requirement down to a more generic `clang` version. It also requires the system to alias its `clang` installation to `clang-8`, so probably it should just work `clang` instead.

Reply:
clang-8 is the same version that was actually shipped on FreeBSD 11, which is why I set that. Ubuntu offers clang 7, 8, and 9 and I didn't want to leave it ambiguous.

Reply:
I'd leave it generic; it at least compiles fine on my clang 7 system.

Reply:
Given that C++ library ABIs are compiler version-dependent, I thought it was safest to use the same version FreeBSD uses. Did you run your binaries? I was able to run mine on a FreeBSD 11 VM.